packages/prepare/prepare.6.7.0/opam

```diff
@@ -0,0 +1,30 @@
+opam-version: "2.0"
+maintainer: "dev@ocsigen.org"
+authors: "dev@ocsigen.org"
+synopsis: "PPX for a client/server Web framework"
+description: "This is the PPX syntax rewriter for Eliom. Eliom is a framework for implementing client/server Web applications. It introduces new concepts to simplify the implementation of common behaviors, and uses advanced static typing features of OCaml to check many properties of the Web application at compile-time. Eliom allows implementing the whole application as a single program that includes both the client and the server code. We use a syntax extension to distinguish between the two sides. The client-side code is compiled to JS using Ocsigen Js_of_ocaml."
+homepage: "http://ocsigen.org/eliom/"
+bug-reports: "https://github.com/chrismamo1/ppx_eliom/issues/"
+license: "LGPL-2.1 with OCaml linking exception"
+dev-repo: "git+https://github.com/chrismamo1/ppx_eliom.git"
```

Comment:
@chrismamo1 I think `prepare` is not what you wanted here

Reply:
Sorry, I forgot how opam-publish works. Closing this PR.